swarm/src/test/resources/docker/swarm.tests.bom

```diff
@@ -396,6 +396,7 @@ brooklyn.catalog:
         swarm.port: 4000
         docker.recovery.stabilizationDelay: 10s
         docker.recovery.failOnRecurringFailuresInThisDuration: 5m
+        swarm.sharedsecuritygroup.create: true
```

Comment:
This is going to be different on differing clouds which we test, do we want to put this here?